cfme/tests/infrastructure/test_providers.py

```diff
@@ -25,6 +26,24 @@
     pytest.mark.provider([InfraProvider], scope="function"),
 ]
 
+discovery_ips = [
+    {'from': ['120', '120', '120', '120'], 'to': '119',
```

Comment:
**REQUIRED:** These addresses are potentially valid, and though we expect to fail, we should use a private address range so that it doesn't potentially hit an external IP.
```10.0.0.0 – 10.255.255.255```
```172.16.0.0 – 172.31.255.255```
```192.168.0.0 – 192.168.255.255```
The 333 ones are OK because even if it didn't catch it was invalid, 333 IS invalid and wouldn't communicate out.